solidity-v1/test/TestPhasedEscrow.js

```diff
@@ -388,6 +392,33 @@ describe("PhasedEscrow", () => {
     assertRewards(baseBalance, transferAmount)
   })
 
+  describe("when withdrawing to a PlainTransferEscrowBeneficiary", () => {
+    const baseBalance = 200000000
+    const transferAmount = 9100000
+
+    before(async () => {
+      phasedEscrow = await PhasedEscrow.new(token.address, { from: owner })
+      const amount = web3.utils.toBN(baseBalance)
+      await token.transfer(phasedEscrow.address, amount, { from: owner })
+
+      rewardsContract = await TestSimpleReceiver.new({ from: owner })
+      rewardsBeneficiary = await PlainTransferEscrowBeneficiary.new(
```

Comment:
```suggestion
      escrowBeneficiary = await PlainTransferEscrowBeneficiary.new(
```

Reply:
It's not going to work because of how `assertRewards` is constructed:
https://github.com/keep-network/keep-core/blob/c5f3acf0658f341fe8656219693b761001485033/solidity-v1/test/TestPhasedEscrow.js#L456